CONTRIBUTING.md

```diff
@@ -322,6 +324,38 @@ doesn't need to wait. A Pull Request may well take longer to be
 merged in. All these precautions are important because Node.js is
 widely used, so don't be discouraged!
 
+### Backporting to Current and LTS Branches
+
+Before backporting or contributing to the LTS releases make sure you are
+familiar with the LTS information and schedules at
+https://github.com/nodejs/LTS#lts-schedule
```

Comment:
`LTS information and schedules at https://github.com/nodejs/LTS#lts-schedule` -> `[LTS information and schedules](https://github.com/nodejs/LTS#lts-schedule).`